packages/fxa-auth-server/lib/types.ts

```diff
@@ -21,6 +21,7 @@ export interface AuthApp extends RequestApplicationState {
   acceptLanguage: string;
   clientAddress: string;
   metricsContext: any;
+  metricsEventUid?: string;
```

Comment:
I think `request.app` is the right place for this in general, though I wonder if it possible/advisable to put the user id in the existing `metricsContext`?

Reply:
Okay yeah upon looking at that more, `stashMetricsContext` expects a token like a `sessionToken`, which we don't have. Not something that can be easily leveraged for our use case.